src/command_line.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+from str2bool import str2bool
+
+from src.providers import providers
+
+__parser = argparse.ArgumentParser()
+
+__parser.add_argument('--concurrency-per-core', '-c', type=int, default=2, dest='concurrency',
+                      help='Number of workers = concurrency * CPU_CORES_COUNT')
+__parser.add_argument('--temp-dir', '-t', type=str, default='./tmp',
+                      help='Base directory to place all downloaded files before bundling')
```

Comment:
I think we should move to mem based FS

Reply:
How often is it updated? How many heavy packages are there?
I can argue that if `pytorch` is so heavy, maybe we shouldn't add it to this tool for now, at least until we know costs of azure resources

Reply:
Because it is downloaded, bundled and then uploaded:
1.  It may be necessary to keep the packages in memory for some parts of this process even if we do use a temp fs
2.  It will be A LOT faster to use mem fs

Reply:
I agree that we might have a problem with Azure resources. In the long term, our code should handle both small and large packages (think about docker images).

about `mem fs` vs `HDD` when we do implement such a feature, we should also introduce a flag for choosing between the two.

Reply:
After pola will introduce a DB, this will be possible. In the meantime, can you agree that implementing tmpfs and folder permissions is premature?

Reply:
I agree

Reply:
Opened as #16

Reply:
@syncush ?

Reply:
resolved